tests/test_ctl.py

```diff
@@ -40,107 +45,196 @@ def test__wait_args_to_opts():
-    assert fake_pgctl.start("data", wait=False).args == "pg_ctl start -D data --no-wait"
-    assert fake_pgctl.start(
+def test_start_cmd(pgctl: ctl.PGCtl) -> None:
+    assert pgctl.start_cmd("data") == shlex.split("pg_ctl start -D data --wait")
+    assert pgctl.start_cmd("data", wait=False) == shlex.split(
+        "pg_ctl start -D data --no-wait"
+    )
+    assert pgctl.start_cmd(
         "data",
         wait=3,
         logfile="logfile",
-    ).args == ("pg_ctl start -D data --wait --timeout=3 --log=logfile")
-    assert fake_pgctl.start("data", k="/tmp/sockets").args == (
+    ) == shlex.split("pg_ctl start -D data --wait --timeout=3 --log=logfile")
+    assert pgctl.start_cmd("data", k="/tmp/sockets") == shlex.split(
         "pg_ctl start -D data --wait -o '-k /tmp/sockets'"
     )
 
 
-def test_stop(fake_pgctl):
-    assert fake_pgctl.stop("data").args == "pg_ctl stop -D data --wait"
-    assert fake_pgctl.stop("data", wait=False).args == "pg_ctl stop -D data --no-wait"
-    assert fake_pgctl.stop(
+def test_stop_cmd(pgctl: ctl.PGCtl) -> None:
+    assert pgctl.stop_cmd("data") == shlex.split("pg_ctl stop -D data --wait")
+    assert pgctl.stop_cmd("data", wait=False) == shlex.split(
+        "pg_ctl stop -D data --no-wait"
+    )
+    assert pgctl.stop_cmd(
         "data",
         wait=3,
         mode="fast",
-    ).args == ("pg_ctl stop -D data --wait --timeout=3 --mode=fast")
+    ) == shlex.split("pg_ctl stop -D data --wait --timeout=3 --mode=fast")
 
 
-def test_restart(fake_pgctl):
-    assert fake_pgctl.restart("data").args == "pg_ctl restart -D data --wait"
-    assert (
-        fake_pgctl.restart("data", wait=False).args
-        == "pg_ctl restart -D data --no-wait"
+def test_restart_cmd(pgctl: ctl.PGCtl) -> None:
+    assert pgctl.restart_cmd("data") == shlex.split("pg_ctl restart -D data --wait")
+    assert pgctl.restart_cmd("data", wait=False) == shlex.split(
+        "pg_ctl restart -D data --no-wait"
     )
-    assert (
-        fake_pgctl.restart(
-            "data",
-            wait=3,
-            mode="fast",
-        ).args
-        == "pg_ctl restart -D data --wait --timeout=3 --mode=fast"
-    )
-    assert fake_pgctl.restart("data", k="/tmp/sockets").args == (
+    assert pgctl.restart_cmd(
+        "data",
+        wait=3,
+        mode="fast",
+    ) == shlex.split("pg_ctl restart -D data --wait --timeout=3 --mode=fast")
+    assert pgctl.restart_cmd("data", k="/tmp/sockets") == shlex.split(
         "pg_ctl restart -D data --wait -o '-k /tmp/sockets'"
     )
 
 
-def test_reload(fake_pgctl):
-    assert fake_pgctl.reload("data").args == "pg_ctl reload -D data"
+def test_reload_cmd(pgctl: ctl.PGCtl) -> None:
+    assert pgctl.reload_cmd("data") == shlex.split("pg_ctl reload -D data")
+
+
+def test_status_cmd(pgctl: ctl.PGCtl) -> None:
+    assert pgctl.status_cmd("data") == shlex.split("pg_ctl status -D data")
+
+
+@pytest.mark.parametrize(
+    "rc, status",
```

Comment:
Can we be more explicit for the variable name and use `return_code` instead of `rc`?
When I read `rc1`, I first thought "Release Candidate 1". OK stupid, I know.